x/clp/handler.go

```diff
@@ -2,6 +2,7 @@ package clp
 
 import (
 	"fmt"
+	clpkeeper "github.com/Sifchain/sifnode/x/clp/keeper"
```

Comment:
I'm a bit hazy on the inclusion of clpkeeper here. Do we not already have the keeper passed as a parameter to the handler?

Reply:
I moved calculations.go to the keeper package .
We have alwasys used the keeper package in handler.go ,just that I had created alias for the functionss ( see alias.go) ,  alias.go and handler.go are both in the same package .

Reply:
I can alias these funtions as well 
keeper.CalculatePoolUnits
keeper.SwapOne

but I am not sure if it makes sense for us to expose them to be used through a direct clp import .

Reply:
If the calculations.go functions are now in the keeper package then can't they be called from the keeper passed to the handler?
 ie: 
 `func handleMsgCreatePool(ctx sdk.Context, keeper Keeper, msg MsgCreatePool)`
 
 I'm not trying to suggest adding them to the alias, I'm just confused about the two instances of the keeper. Is the clpKeeper different from the keeper passed as a parameter above?

Reply:
The calculations.go were moved to the keeper package , but they not part part of the keeper struct .

Reply:
Okay gotcha, that answers my question. Sorry for the confusion.